<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Anh-IT-ngheo-2.local" failures="12" tests="12" name="Command line test" time="1.097" errors="0" timestamp="2024-07-01T02:33:25 ICT">
  <testcase classname="Tests.ProductLineTest" name="createProductLine" time="0.653">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at Tests.ProductLineTest.createProductLine(ProductLineTest.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- createProductLine -->
  <testcase classname="Tests.ProductLineTest" name="deleteProductLine" time="0.032">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at Tests.ProductLineTest.deleteProductLine(ProductLineTest.java:94)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- deleteProductLine -->
  <testcase classname="Tests.ProductLineTest" name="getListProductLineDefault" time="0.026">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at Tests.ProductLineTest.getListProductLineDefault(ProductLineTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- getListProductLineDefault -->
  <testcase classname="Tests.ProductLineTest" name="updateProductLine" time="0.03">
    <failure type="java.lang.AssertionError" message="expected [201] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [401]
at Tests.ProductLineTest.updateProductLine(ProductLineTest.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- updateProductLine -->
  <testcase classname="Tests.BrandTest" name="createBrand" time="0.0">
    <failure type="org.testng.TestNGException" message="
Cannot inject @Test annotated Method [createBrand] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [createBrand] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- createBrand -->
  <testcase classname="Tests.BrandTest" name="deleteBrand" time="0.023">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at Tests.BrandTest.deleteBrand(BrandTest.java:120)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- deleteBrand -->
  <testcase classname="Tests.BrandTest" name="getListBrandDefault" time="0.189">
    <failure type="java.lang.IllegalArgumentException" message="accessToken cannot be null">
      <![CDATA[java.lang.IllegalArgumentException: accessToken cannot be null
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.common.assertion.AssertParameter.notNull(AssertParameter.groovy:26)
at io.restassured.internal.common.assertion.AssertParameter$notNull.callStatic(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:54)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:217)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:240)
at io.restassured.internal.AuthenticationSpecificationImpl.oauth2(AuthenticationSpecificationImpl.groovy:158)
at io.restassured.specification.AuthenticationSpecification$oauth2.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:50)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)
at io.restassured.internal.AuthenticationSpecificationImpl.oauth2(AuthenticationSpecificationImpl.groovy:151)
at APIs.BrandAPIs.getListBrands(BrandAPIs.java:19)
at Tests.BrandTest.getListBrandDefault(BrandTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- getListBrandDefault -->
  <testcase classname="Tests.BrandTest" name="updateBrand" time="0.03">
    <failure type="java.lang.AssertionError" message="
Expected: is &amp;lt;200&amp;gt;
     but: was &amp;lt;404&amp;gt;">
      <![CDATA[java.lang.AssertionError: 
Expected: is <200>
     but: was <404>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at Tests.BrandTest.updateBrand(BrandTest.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- updateBrand -->
  <testcase classname="Tests.ModelTest" name="createModel" time="0.027">
    <failure type="java.lang.AssertionError" message="expected [201] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [404]
at Tests.ModelTest.createModel(ModelTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- createModel -->
  <testcase classname="Tests.ModelTest" name="deleteModel" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [201] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [404]
at Tests.ModelTest.deleteModel(ModelTest.java:96)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- deleteModel -->
  <testcase classname="Tests.ModelTest" name="getListModelDefault" time="0.025">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at Tests.ModelTest.getListModelDefault(ModelTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- getListModelDefault -->
  <testcase classname="Tests.ModelTest" name="updateModel" time="0.025">
    <failure type="java.lang.AssertionError" message="expected [201] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [404]
at Tests.ModelTest.updateModel(ModelTest.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- updateModel -->
  <testcase classname="APIs.BaseSetup" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="APIs.BaseSetup" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
</testsuite> <!-- Command line test -->
